fastlane/lib/fastlane/actions/set_build_number_repository.rb

```diff
@@ -13,7 +13,10 @@ def self.run(params)
           use_hg_revision_number: params[:use_hg_revision_number]
         )
 
-        Fastlane::Actions::IncrementBuildNumberAction.run(build_number: build_number)
+        Fastlane::Actions::IncrementBuildNumberAction.run(
+          build_number: build_number,
+          xcodeproj: params[:xcodeproj]
```

Comment:
not that familiar with ruby, not sure if I should be doing an if check to make sure `params[:xcodeproj]` isn't `nil` before passing it in.